### ISSUE TITLE
Properly model NUSIZ during missile decode and draw

### ISSUE THREAD

**Issue by DirtyHairy** [stella-emu/stella#93]

**Comment by thrust26** [comment]
@DirtyHairy Is this still an issue?

**Comment by DirtyHairy** [comment]
Yes. This issue is basically a reminder for me to revisit NUSIZ changes during missile decode and draw and refine the emulation. However, this is an exotic edge case that has not caused any issues so far, so I don't see this as high priority.
